Sources/GPX/OAGPXDatabase.mm

```diff
@@ -415,8 +415,11 @@ - (void)load
         BOOL didAddFiles = [self addNewGpxFiles:existingGpxPaths];
         if (didAddFiles)
         {
-            [[NSNotificationCenter defaultCenter] postNotificationName:kNotificationNewTracksFetched
-                                                                object:self];
+            [self save];
+        }
+        else
+        {
+            [[NSNotificationCenter defaultCenter] postNotificationName:kGPXDBTracksReloaded object:self];
```

Comment:
Looks suspicious. Why `save` is needed? Why kGPXDBTracksReloaded if not `didAddFiles`?